docs/get-started/using-with-react.md

```diff
@@ -1,20 +1,18 @@
 # Using deck.gl with React
 
-deck.gl works perfectly for [React](https://facebook.github.io/react/)-based applications. All deck.gl layers fit naturally into React's component render flow.
+While not directly based on React, deck.gl is designed from ground up to work with [React](https://facebook.github.io/react/)-based applications. deck.gl layers fit naturally into React's component render flow and flux/redux based appications. deck.gl layers will be performantly rerendered whenever you rerender your normal JSX or React components.
```

Comment:
The link to the react doc should be put to the first occurrence of React, but I'm sure people that go that far in the docs know what React is anyway